packages/commercetools/theme/components/Checkout/ShippingForm.vue

```diff
@@ -284,10 +284,10 @@ export default {
       return Boolean(addresses?.length);
     });
 
-    const handleStepSubmit = () => context.emit('stepSubmit');
+    const handleStepSubmit = () => context.emit('submit');
     const handleAddressSubmit = (reset) => async () => {
       const addressId = currentAddressId.value;
-      await props.handleShippingAddressSubmit(shippingDetails.value);
+      await props.handleShippingAddressSubmit({ shippingDetails: shippingDetails.value });
       if (addressId !== NOT_SELECTED_ADDRESS && setAsDefault.value) {
```

Comment:
+ its redundant to assign a const into an empty string

Reply:
I will remove this part from this PR, as https://github.com/vuestorefront/vue-storefront/pull/5564 takes care of that